src/utils/apv.ts

```diff
@@ -27,7 +25,9 @@ function analyzeApvFromToken(token: string): IApv {
     raw: token,
     version: parseInt(version),
     signer: signer.replace("0x", ""),
-    signature: Buffer.from(signature, "base64").toString("hex"),
+    signature: Buffer.from(signature.replaceAll(".", "/"), "base64").toString(
```

Comment:
Though I cannot reference the above code not changed in this pull request, the `const extra = decode(Buffer.from(rawExtra, "base64"))!;` part also should be fixed by replacing `'.'` with `'/'` like signature does.

Reply:
[It seems so](https://github.com/planetarium/libplanet/blob/e9ba50938e53aaa87fcf3d3ec943201b191b21e0/Libplanet.Net/AppProtocolVersion.cs#L97-L101). thanks for pointing out.